obspy/clients/iris/client.py

```diff
@@ -583,34 +583,41 @@ def distaz(self, stalat, stalon, evtlat, evtlon):
         Latitudes are converted to geocentric latitudes using the WGS84
         spheroid to correct for ellipticity.
 
+        The distance (in degrees) and distancemeters are the distance
```

Comment:
Can you put the two names in quotes or double backticks, so it's clear that `distancemeters` is not a typo?